src/cim/components/Agent/BMCForm.tsx

```diff
@@ -124,7 +125,7 @@ const getValidationSchema = (hasDHCP: boolean, usedHostnames: string[], origHost
   Yup.object({
     name: Yup.string().required(),
     hostname: hostnameValidationSchema(origHostname, usedHostnames),
-    bmcAddress: Yup.string().required(),
+    bmcAddress: bmcAddressValidationSchema,
```

Comment:
BMC address is required field to correctly register bare metal host.
```suggestion
    bmcAddress: bmcAddressValidationSchema.required(),
```

Reply:
Ok. I add this change